packages/web/.env.development

```diff
@@ -0,0 +1 @@
+GATSBY_GOOGLE_API_KEY=AIzaSyCd7aKSM3Yj1anzUZvJBv6ui1WjzOfnDs4
```

Comment:
This key is visible on the client side browser but it's best not to keep it in the repository.  You can remove it from the repository with `git rm --cached file` (which doesn't delete your local copy) and avoid commiting in the future it by adding the .env.* file to it to .gitignore.

Reply:
Done

Reply:
Did you commit and push?

Reply:
just now